vehicle/ford.go

```diff
@@ -29,9 +29,11 @@ func NewFordFromConfig(other map[string]interface{}) (api.Vehicle, error) {
 	cc := struct {
 		embed               `mapstructure:",squash"`
 		User, Password, VIN string
+		Domain              string
 		Cache               time.Duration
 	}{
-		Cache: interval,
+		Domain: "de",
```

Comment:
```suggestion
		Domain: "com",
```

Reply:
how to set the domain in the config file as with "com" it's not working for me